src/libraries/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -41,15 +43,15 @@ public static TimeFunction Create(SetTime setter, GetTime getter, DateTimeKind k
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/40528", TestPlatforms.Browser)]
         public void SettingUpdatesProperties()
         {
             T item = GetExistingItem();
 
             Assert.All(TimeFunctions(requiresRoundtripping: true), (function) =>
             {
                 // Checking that milliseconds are not dropped after setter.
-                DateTime dt = new DateTime(2014, 12, 1, 12, 3, 3, isHFS ? 0 : 321, function.Kind);
+                // Emscripten drops milliseconds in Browser
+                DateTime dt = new DateTime(2014, 12, 1, 12, 3, 3, (isHFS || isBrowser) ? 0 : 321, function.Kind);
```

Comment:
I think it would be better to directly use `PlatformDetection.IsBrowser` here.

Reply:
It looks like there will be a couple of other tests that will run into similar issues re: granularity like HFS

Reply:
Maybe we can add the variable when those other tests do call `PlatformDetection.IsBrowser`?

Reply:
I was going to start fixing those tests too, so I feel like it makes more sense to keep it in?